src/master-playlist-controller.js

```diff
@@ -260,6 +260,9 @@ export class MasterPlaylistController extends videojs.EventTarget {
     this.useCueTags_ = useCueTags;
     this.blacklistDuration = blacklistDuration;
     this.enableLowInitialPlaylist = enableLowInitialPlaylist;
+    this.goalBufferLength_ = Config.GOAL_BUFFER_LENGTH;
```

Comment:
Refactor to use the properties on `Hls` (see [this](https://github.com/videojs/videojs-contrib-hls/blob/7186be0866f22a2a1e6234de1d53b1915e065507/src/videojs-contrib-hls.js#L49-L74))